packages/it-flat-batch/index.js

```diff
@@ -1,12 +1,23 @@
 'use strict'
 
-async function * batch (source, size) {
-  size = parseInt(size)
+/**
+ * Takes an (async) iterable that emits variable length arrays of things and
+ * returns an async iterable that emits those thnigs in fixed-size batches.
+ *
+ * @template T
+ * @param {AsyncIterable<T[]>|Iterable<T[]>|AsyncIterable<T>|Iterable<T>} source
```

Comment:
I just looked at code which did `arr.concat(t)` and thought it could be either, but following does make more sense 

```suggestion
 * @param {AsyncIterable<T[]>|Iterable<T[]>} source
```